ci/deps/pyspark.yml

```diff
@@ -1 +1 @@
-pyspark>=3.1.1
```

Comment:
hmm are we still testing the same as before? cc @icexelloss

Reply:
Yes, the pinning is the same for the min, and this was only for speedup. I can leave it if you want, but it's installing the latest version as expected:
```
pyspark                                      3.1.1  pyh44b312d_0
```